wordle-web/src/App.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import { RouterLink, RouterView } from 'vue-router'
+import NavBar from '@/components/NavBar.vue'
+</script>
+
+<template>
+  <div class="main">
```

Comment:
To further build this out in the future and to replace the default vue stuff, I recommend refactoring the main template to utilize the grid system from vuetify. The grid system is shared among nearly all UI libraries, and there are many examples in the vuetify docs. It's an easy way to quickly throw together content layouts.
https://vuetifyjs.com/en/components/grids/